src/ITILValidationTemplate_Target.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+class ITILValidationTemplate_Target extends CommonDBRelation
+{
+    public $dohistory = true;
+
+    public static $itemtype_1 = 'ITILValidationTemplate';
+    public static $items_id_1 = 'itilvalidationtemplates_id';
+    public static $take_entity_1 = false;
+
+    public static $itemtype_2 = 'itemtype';
+    public static $items_id_2 = 'items_id';
+    public static $take_entity_2 = true;
+
+    public static function getTypeName($nb = 0)
+    {
+        return _n('Validation template target', 'Validation template targets', $nb);
+    }
+
+    public static function getTargets($itilvalidationtemplates_id)
```

Comment:
Method could probably be simplified to `$itiltargets->find()` result directly.
Indeed, itemtype could then be retrieved using `ITILValidationTemplate_Target::getTargets()[0]['itemtype']`.